tests/functional/firefox/new/test_download.py

```diff
@@ -22,14 +19,14 @@ def test_download_button_displayed(base_url, selenium):
 @pytest.mark.skip_if_internet_explorer(reason='https://github.com/SeleniumHQ/selenium/issues/448')
 @pytest.mark.nondestructive
 def test_click_download_button(base_url, selenium):
-    page = DownloadPage(selenium, base_url, params='?v=a').open()
+    page = DownloadPage(selenium, base_url).open()
     thank_you_page = page.download_firefox()
     assert thank_you_page.seed_url in selenium.current_url
 
 
 @pytest.mark.nondestructive
 def test_other_platforms_modal(base_url, selenium):
```

Comment:
Pretty sure this will fail on the new page since we no longer have a modal, right?

Reply:
✅  Added the modal back.